sql/004_create_NGOs.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE `non_profit_org` (
+  `n_name` varchar(45) NOT NULL,
+  `n_city` varchar(45) NOT NULL,
+  `n_services_offered` varchar(45) NOT NULL,
+  `n_telephone` varchar(45) NOT NULL,
+  `number_of_volunteers` int NOT NULL,
+  PRIMARY KEY (`n_name`)
```

Comment:
What if the name changes? Using synthetic primary keys (`id not null auto_increment`) would be much better.